src/main/java/com/exasol/projectkeeper/validators/dependencies/DependenciesBrokenLinkReplacer.java

```diff
@@ -0,0 +1,32 @@
+package com.exasol.projectkeeper.validators.dependencies;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.exasol.projectkeeper.BrokenLinkReplacer;
+
+public class DependenciesBrokenLinkReplacer {
```

Comment:
Do we still write Javadocs or rules were changed?

Reply:
I just missed it.